config/default_jobs.yaml

```diff
@@ -1,5 +1,5 @@
 define:
-  min_year: &min_year 2022
+  min_year: &min_year 2023
```

Comment:
Sneaking these changes in. If we want to match the current sqoop behavior we actually want to grab >= 2023 data.